job_templates/snippet/publisher_warnings_ng.xml.em

```diff
@@ -0,0 +1,57 @@
+<io.jenkins.plugins.analysis.core.steps.IssuesRecorder plugin="warnings-ng@7.2.2">
```

Comment:
In an `em` template the literal `@` needs to be escape with another `@`. Currently this should result in an error (have you tried to generate the job xml using this patch?).

Same below.

Reply:
Thank you. No I haven't tried generating the job.